src/main/java/browserview/BrowserComponent.java

```diff
@@ -204,9 +204,11 @@ public void showIssue(String repoId, int id, boolean isPullRequest, boolean isFo
         if (isPullRequest) {
             logger.info("Showing pull request #" + id);
             runBrowserOperation(() -> driver.get(GitHubURL.getPathForPullRequest(repoId, id), isForceRefresh));
+            runBrowserOperation(() -> driver.scrollToBottom());
         } else {
             logger.info("Showing issue #" + id);
             runBrowserOperation(() -> driver.get(GitHubURL.getPathForIssue(repoId, id), isForceRefresh));
+            runBrowserOperation(() -> driver.scrollToBottom());
```

Comment:
Since they occur in both cases, these should be outside the conditional. Also, a direct method reference can be used.